sites/skeleton.dev/src/content/docs/components/accordion/meta.mdx

```diff
@@ -1,9 +1,10 @@
 ---
 title: Accordion
 description: Divide content into collapsible sections.
-srcSvelte: '/src/components/Accordion'
-srcReact: '/src/components/Accordion'
+srcSvelte: '/src/components-composed/accordion'
```

Comment:
If we're taking an axe to the v3 component docs and lib/components per each framework package, let's make sure we also remove any reference to `-composed`. That was only needed for the v4 preview on the v3 branch. Going forward the new components should just take the place of the old components - both in terms of packages and docs.